fiscalyear.py

```diff
@@ -24,12 +24,8 @@
 START_DAY = 1
 
 
-@contextlib.contextmanager
-def fiscal_calendar(start_year=None,
-                    start_month=None,
-                    start_day=None):
-    """A context manager that lets you modify the start of the fiscal calendar
-    inside the scope of a with-statement.
+def validate_fiscal_calendar_params(start_year, start_month, start_day):
```

Comment:
Can you add an underscore to the start of this function name? I don't want it to be user-facing and end up in the API docs.